datafusion/sqllogictest/test_files/select.slt

```diff
@@ -1473,7 +1473,7 @@ DROP TABLE t;
 
 # related to https://github.com/apache/datafusion/issues/8814
 statement ok
-create table t(x int, y int) as values (1,1), (2,2), (3,3), (0,0), (4,0);
+create table t(x bigint, y bigint) as values (1,1), (2,2), (3,3), (0,0), (4,0);
```

Comment:
Could you please keep the existing test and add a new test that uses `bigint` so it is clearer what behavior is changing?